src/Compilers/CSharp/Portable/Symbols/TypeSymbol.cs

```diff
@@ -1106,6 +1115,24 @@ private static void ReportImplicitImplementationMatchDiagnostics(Symbol interfac
                     }
                 }
             }
+
+            if (implicitImpl.ContainingType.IsInterface && implementingType.ContainingModule != implicitImpl.ContainingModule)
+            {
+                // PROTOTYPE(DefaultInterfaceImplementation): Should we check language version as well?
+                // Usually, it is done based on specific syntax that targets a new feature, but in this case
```

Comment:
> I think we use language version in binding, in rare cases.

That is true, but it still in response to specific syntax that wouldn't be valid for prior version of the language. In this case, there is no special syntax used.